usaspending_api/search/delta_models/subaward_search.py

```diff
@@ -0,0 +1,558 @@
+            ON aa.id = a.awarding_agency_id
+    LEFT OUTER JOIN
+        global_temp.toptier_agency AS taa
+            ON taa.toptier_agency_id = aa.toptier_agency_id
+    LEFT OUTER JOIN
+        global_temp.subtier_agency AS saa
+            ON saa.subtier_agency_id = aa.subtier_agency_id
+    LEFT OUTER JOIN
+        global_temp.agency AS fa
+            ON fa.id = a.funding_agency_id
+    LEFT OUTER JOIN
+        global_temp.toptier_agency AS tfa
+            ON tfa.toptier_agency_id = fa.toptier_agency_id
+    LEFT OUTER JOIN
+        global_temp.subtier_agency AS sfa
+            ON sfa.subtier_agency_id = fa.subtier_agency_id
+    LEFT OUTER JOIN
+        (
+            SELECT
+                faba.award_id,
+                SORT_ARRAY(COLLECT_SET(DISTINCT CAST(taa.treasury_account_identifier AS INTEGER))) AS treasury_account_identifiers
+            FROM
+                global_temp.treasury_appropriation_account AS taa
+            INNER JOIN
+                raw.financial_accounts_by_awards AS faba
+                    ON taa.treasury_account_identifier = faba.treasury_account_id
+            WHERE
+                faba.award_id IS NOT NULL
+            GROUP BY
+                faba.award_id
+        ) AS tas
+            ON (tas.award_id = a.id)
+    LEFT OUTER JOIN
+        recipient_summary AS recipient_lookup
+            ON (recipient_lookup.uei = UPPER(bs.sub_awardee_or_recipient_uei)
+                AND bs.sub_awardee_or_recipient_uei IS NOT NULL AND recipient_lookup.row = 1)
+    LEFT OUTER JOIN
+        recipient_summary AS parent_recipient_lookup
+            ON (parent_recipient_lookup.uei = UPPER(bs.sub_ultimate_parent_uei)
+                AND bs.sub_ultimate_parent_uei IS NOT NULL AND parent_recipient_lookup.row = 1)
+    LEFT OUTER JOIN
+        location_summary AS pop
+            ON (pop.feature_name = UPPER(bs.sub_place_of_perform_city_name)
+                AND pop.state_alpha = UPPER(bs.sub_place_of_perform_state_code)
+                AND pop.row_num = 1)
+    LEFT OUTER JOIN
+        location_summary AS rec
+            ON (rec.feature_name = UPPER(bs.sub_legal_entity_city_name)
+                AND rec.state_alpha = UPPER(bs.sub_legal_entity_state_code)
+                AND rec.row_num = 1)
+    LEFT OUTER JOIN
+        global_temp.ref_country_code AS pcc
+            ON (pcc.country_code = UPPER(bs.sub_place_of_perform_country_co)
+                AND bs.sub_place_of_perform_country_co IS NOT NULL)
+    LEFT OUTER JOIN
+        global_temp.ref_country_code AS rcc
+            ON (rcc.country_code = UPPER(bs.sub_legal_entity_country_code)
+                AND bs.sub_legal_entity_country_code IS NOT NULL)
+    LEFT OUTER JOIN
+        global_temp.psc
+            ON fpds.product_or_service_code = psc.code
+    LEFT OUTER JOIN
+        global_temp.references_cfda AS cfda
+            ON cfda.program_number = split(bs.cfda_numbers, ',')[0]
+    WHERE bs.subaward_number IS NOT NULL
```

Comment:
Might be useful to include sql comment inline, as to why this condition is needed/helpful.